### ISSUE TITLE
Hex.toString truncates hex values that begin with a "0"

### ISSUE THREAD

**Issue by supermacro** [rtfeldman/elm-hex#2]
Demo: https://ellie-app.com/bcpSncpmBGSa1

I wouldn't call this a bug ... but I'll share anyways.

I do some hex math in a project of mine and then after my hex math, I convert these hex values into strings (to be used for CSS colors). But I noticed that `Hex.toString` is sometimes producing hex values that cannot be used for CSS colors (the hex digits must be either 6 or 3 digits to be used for CSS colors from my understanding) when the hexadecimal value begins with one or more 0s (see ellie demo for an example).

*tldr:* all leading 0's are truncated, which might produce unpredictable results to those who are using this package for CSS colors.

**Comment by supermacro** [comment]
This ended up being my workaround for folks who need to guarantee 6-digit hex strings:

```elm
-- produces 6-digit CSS hex values
-- to ensure that CSS engines can properly interpret the
-- hexadecimal as a RGB color
toCssColorValue : Int -> String
toCssColorValue val =
    let
        -- this value is going to be between 6 and 1 digits
        rawString = Hex.toString val

        length = String.length rawString
        
        missingZeroes = String.repeat (6 - length) "0"
    in
    "#" ++ missingZeroes ++ rawString
```

Hope this helps others!
